apps/docs/src/.vitepress/components/OnyxShadowVariables.vue

```diff
@@ -12,13 +12,7 @@ const shadows = [
 </script>
 
 <template>
-  <DesignVariableCard
-    v-for="shadow in shadows"
-    :key="shadow"
-    :name="shadow"
-    class="variable"
-    hide-value
-  >
+  <DesignVariableCard v-for="shadow in shadows" :key="shadow" :name="shadow" class="variable">
```

Comment:
I guess the `hide-value` property is now unused so we can remove it from the `DesignVariableCard`